sc-memory/sc-memory/src/scs/scs_parser.cpp

```diff
@@ -546,14 +645,32 @@ void Parser::ProcessContourEnd(ElementHandle const & contourHandle)
     newElements.insert(t.m_target);
   }
 
-  for (auto const & el : newElements)
+  std::unordered_set<ElementID> addedElementTypesIntoStructure;
```

Comment:
probably something in this method causes core to be dumped during kb build(I've added second sc_node_tuple)
![image](https://github.com/user-attachments/assets/e93f4bbc-998e-458b-a494-f456d4c84baf)
![image](https://github.com/user-attachments/assets/580a2b30-3ac9-42d6-89b5-56cc6589202a)

Reply:
I renamed this template to sc_node and log had new type missing
.gwf - failed
Can't extend type `VarCommonArc` to type `` for element `..connector_118`.
Clean state...

Reply:
it still does not print types
failed
Can't extend type `ConstPermPosArc` to type `` for element `..connector_133`, because `` is not subtype of `ConstPermPosArc`.